test/e2e/config/default/controller_manager_config.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: config.kueue.x-k8s.io/v1beta1
 kind: Configuration
 metrics:
   enableClusterQueueResources: true
+  localQueueMetrics:
+    localQueueSelector: {}
```

Comment:
If we want to verify this behavior in E2E, we should use customconfig E2E.

Reply:
There are existing LoC which test the behavior in the e2e tests. I've added this here because without it, the feature will default to off